mlflow/pyfunc/__init__.py

```diff
@@ -923,6 +923,12 @@ def _parse_spark_datatype(datatype: str):
     return udf(lambda x: x, returnType=datatype).returnType
 
 
+def _is_none_or_nan(value):
+    # if value is not numeric type, `np.isnan(value)` raises error,
+    # so we add condition `isinstance(value, float)` before it.
+    return value is None or isinstance(value, float) and np.isnan(value)
```

Comment:
/ai simplify the comment in the following code.

Reply:
/ai simplify the comment in the following code. Please make sure to mention why `isinstance(value, float)` is needed.